.gitignore

```diff
@@ -23,3 +23,5 @@ plugins
 gh-pages
 .vscode/ipch
 dev-packages/electron/compile_commands.json
+*.tsbuildinfo
```

Comment:
Are these files indexes produced by tsc and eslint or it is not recommended?

Reply:
They are indeed indexes/caches. But they shouldn't be check in the project, as it is specific to each local clone/build.

Reply:
ok